src/interfaces.ts

```diff
@@ -10,13 +10,17 @@ export interface LibraryData {
     short_name: string,
     timestamp?: string,
   };
-  urls_and_contact?: {
+  urls_and_contact: {
     authentication_url: string,
     contact_email: string,
     opds_url: string,
     web_url: string,
     validated?: string
   };
+  areas: {
```

Comment:
I think `areas` can also be optional if both its attributes are also optional.

Reply:
The library will always have an `areas` value (see comment re: filter, up top!).  If its focus/service areas aren't specified, then `library[areas]` will be `{focus: [], service: []}`.  (As I type this, I realize that this means that those attributes aren't actually optional; hmm...).  But `areas` itself is always there.

Reply:
Yeah, that makes sense. But can the server return `areas: null`?

Reply:
No--will always have `focus` and `service`.